### PR TITLE
Update live tests

### PR DESCRIPTION
**Pull request by alibazlamit**
@edevenport Added the missing live tests in this PR

### CODE REVIEW

File: profitbricks/client.py [hunk 1]
```diff
@@ -2315,3 +2315,33 @@ def __repr__(self):
...
+        """
+        self.name = name
+        self.description=description
+        self.size = int(size)
+        self.licence_type = licence_type
+        self.location=location
+
+    def __repr__(self):
+        return (('<Snapshot: name=%s, description=%s, size=%s,location=%s, ...>')
+                % (self.name, str(self.size), self.description,
```
Comment (bdrung) [inline profitbricks/client.py]: Better use the format() method.

File: tests/test_firewall_live.py [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+        fwrule2.port_range_end = 8080
+        fwrule2.name = "8080"
+        self.fwrule2 = self.client.create_firewall_rule(
+            datacenter_id=self.datacenter['id'],
+            server_id=self.server['id'],
+            nic_id=self.nic1['id'],
+            firewall_rule=fwrule2)
+        wait_for_completion(self.client, self.fwrule2, 'create_fwrule2')
+
+    @classmethod
```
Comment (bdrung) [inline tests/test_firewall_live.py]: Wrong line ending

File: tests/test_loadbalancer_live.py [hunk 3]
```diff
@@ -0,0 +1,175 @@
+import unittest
+
+from helpers import configuration
+from helpers.resources import resource, wait_for_completion
+from profitbricks.client import ProfitBricksService
+from profitbricks.client import Datacenter, LoadBalancer, LAN, NIC, Server
+
+
+class TestLoadBalancer(unittest.TestCase):
+    @classmethod
```
Comment (bdrung) [inline tests/test_loadbalancer_live.py]: Wrong line ending

File: tests/test_loadbalancer_live.py [hunk 4]
```diff
@@ -0,0 +1,175 @@
...
+
+        # Associate nic to loadbalancer3
+        self.associated_nic3 = self.client.add_loadbalanced_nics(
+            datacenter_id=self.datacenter['id'],
+            loadbalancer_id=self.loadbalancer3['id'],
+            nic_id=self.nic1['id'])
+
+        wait_for_completion(self.client, self.associated_nic3, 'associate_nic3')
+
+    @classmethod
```
Comment (bdrung) [inline tests/test_loadbalancer_live.py]: Wrong line ending

File: tests/test_server_live.py [hunk 5]
```diff
@@ -1,11 +1,15 @@
 import unittest
 
+import time
```
Comment (bdrung) [inline tests/test_server_live.py]: Standard import should be in the first import group.

File: tests/test_server_live.py [hunk 6]
```diff
@@ -33,6 +37,8 @@ def setUpClass(self):
             volume=Volume(**self.resource['volume']))
         wait_for_completion(self.client, self.volume2, 'create_volume')
 
+
+
```
Comment (bdrung) [inline tests/test_server_live.py]: Useless newline (pep8 won't be happy)

**Comment by alibazlamit** [comment]
Thanks @bdrung, i have updated the code with the changes requested.

**Pull request merged**